pkg/bgpv1/manager/reconcilerv2/policies_test.go

```diff
@@ -483,8 +484,25 @@ func Test_MergeRoutePolicies(t *testing.T) {
 				req.NoError(err)
 			}
 
+			if tt.expected != nil {
+				req.NotNil(result)
+				SortRouteStatementsByName(result.Statements)
+				SortRouteStatementsByName(tt.expected.Statements)
+			}
+
 			req.Equal(tt.expected, result)
 		})
 	}
+}
 
+func SortRouteStatementsByName(statements []*types.RoutePolicyStatement) {
+	slices.SortFunc(statements, func(i, j *types.RoutePolicyStatement) int {
+		if i.Conditions.String() < j.Conditions.String() {
+			return -1
+		} else if i.Conditions.String() > j.Conditions.String() {
+			return 1
+		} else {
+			return 0
+		}
+	})
 }
```

Comment:
nitpick, feel free to ignore

```suggestion
func SortRouteStatementsByName(statements []*types.RoutePolicyStatement) {
	slices.SortFunc(statements, func(i, j *types.RoutePolicyStatement) int {
		return cmp.Compare(i.Conditions.String(), j.Conditions.String())
	})
}
```

Reply:
I'll go ahead and merge the PR to fix CI. Let's clean this up later? I think this suggestion is valid.

Reply:
Sorry just checking this PR ( timezones ). Here is the follow up 
https://github.com/cilium/cilium/pull/36888

Reply:
agreed, I love to merge it in asap as well, that is why _feel free to ignore_.